delphi-frontend/src/main/antlr3/au/com/integradev/delphi/antlr/Delphi.g

```diff
@@ -1447,7 +1457,7 @@ Alpha                   : 'a'..'z'
                         // surrogate pairs
                         | ('\uD800'..'\uDBFF') ('\uDC00'..'\uDFFF')
                         ;
-fragment
+  fragment
```

Comment:
What's the reason for this indent?

Reply:
Oops, apparently didn't push the fix for this (or the comments for the weird new parser rules)

Reply:
That's fixed now.